src/components/lime-ai-avatar/lime-ai-avatar.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, h, Prop } from '@stencil/core';
+
+/**
+ * This component displays an avatar, representing Lime AI assistants.
+ * :::warning
+ * This is a private component used internally in the Lime's various applications,
+ * which is the reason for having it in Lime Elements —to ease the distribution
+ * of the component across all our apps.
+ *
+ * 3rd party developers are not allowed use this component directly.
+ * :::
```

Comment:
I was just about to ask when I saw this… should we _really_ put this component in lime-elements? We will presumably have to update the license file to explicitly exclude this component from the license that applies to the rest of the code.

Also, technically, by pushing this branch with under the current license, I think we've technically released this for public use now… so… 😕

Reply:
I don't know. I can't think of another way of distributing this internally for other Lime apps. And I think it's not a big deal if other 3rd party developers want to use this component. But of course we can leave a disclaimer and ask them not to. This could be only seen as a matter of trademark and branding. Unless we trademark our AI Avatar (Like what we do for Lime Logos), we can't really stop anyone not to use this as their own AI avatar. Right? 

Anyhow, I think if people care about their products, they'd need to come up with something unique for themselves. Personally, I don't want to be too protectionist here.

If you think we can limit this to the CRM, we could move it to the Building Blocks repo.

Reply:
Yeah, OK. Sounds like this is good enough then 👍